src/main/java/mekanism/common/tile/base/TileEntityMekanism.java

```diff
@@ -738,6 +783,10 @@ public CompoundTag getReducedUpdateTag() {
             component.addToUpdateTag(updateTag);
         }
         updateTag.putFloat(NBTConstants.RADIATION, radiationScale);
+        // Sync the name with clients
+        if (this.customName != null && isNameable()) {
+            updateTag.putString(NBTConstants.CUSTOM_NAME, Component.Serializer.toJson(this.customName));
+        }
```

Comment:
What is the use of syncing this stuff in the update tag? (Especially after the changes I made yesterday). Is it just so if a mod calls the methods on the client they have results, given I am honestly not quite sure how much that matters.

Reply:
I'm not sure if the syncing is needed anymore. Let me see 🤔